deis-workflow-manager/manifests/deis-workflow-manager-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: deis-workflow-manager
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  type: LoadBalancer
```

Comment:
Should we prefer exposing this through the router as a "routable application" rather than giving it its own external load balancer-- to which there is a cost associated on most platforms?